pkg/v3ctl/stream/createrecord.go

```diff
@@ -60,6 +60,13 @@ func newCreateRecordCommandeer(createStreamCommandeer *createStreamCommandeer) (
 				return errors.Wrap(err, "Failed to get create stream")
 			}
 
+			putRecordsResponse, ok := response.Output.(v3io.PutRecordsOutput)
+			if ok && putRecordsResponse.FailedRecordCount != 0 {
+				return errors.Wrap(err,
```

Comment:
You're wrapping `err` here, but it'll be nil. You need to return a new error

Reply:
fixed